src/box/box.cc

```diff
@@ -1515,6 +1543,13 @@ box_update_replication_synchro_quorum(void)
 	if (quorum <= 0 || quorum >= VCLOCK_MAX)
 		panic("failed to eval/fetch replication_synchro_quorum");
 
+	/**
```

Comment:
Inside of functions we usually start comments with `/*` instead of `/**`.

Reply:
Fixed